<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messageDialog</class>
 <widget class="QWidget" name="messageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>159</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>B Yekan</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::RightToLeft</enum>
  </property>
  <widget class="QLabel" name="messageText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
